lib/beaker/dsl/install_utils/pe_defaults.rb

```diff
@@ -32,7 +32,7 @@ module PEDefaults
             'puppetbin'        => '/opt/puppet/bin/puppet',
             'puppetbindir'     => '/opt/puppet/bin',
             'puppetsbindir'    => '/opt/puppet/sbin',
-            'privatebindir'    => '/opt/puppetlabs/puppet/bin',
+            'privatebindir'    => '/opt/puppet/bin',
```

Comment:
after talking with John, we realized that this is the correct path, but that other logic was causing the wrong hosts to be set with their paths, so we've updated that instead of this.